apps/docs/content/guides/database/extensions/pgaudit.mdx

```diff
@@ -41,68 +46,389 @@ To disable an extension you can call `drop extension`.
+| misc     | Less common commands (FETCH, CHECKPOINT)                              | Captures obscure actions for deeper analysis if needed.                    |
+| all      | Everything above                                                      | Comprehensive logging for complete audit trails.                           |
 
-For a full list of available settings see [settings docs](https://github.com/pgaudit/pgaudit/blob/master/README.md#settings). Be aware that the `all` setting will generate a very large volume of logs.
+Below is a limited example of how to assign pgaudit to monitor specific categories.
+
+```sql
+-- log all CREATE, ALTER, and DROP events
+... pgaudit.log = 'ddl';
+
+-- log all CREATE, ALTER, DROP, and SELECT events
+... pgaudit.log = 'read, ddl';
+
+-- log nothing
+... pgaudit.log = 'none';
+```
+
+### Session Logging
+
+When you are connecting in a session environment, such as a PSQL connection, you can configure pgaudit to record events initiated within the session.
 
 <Admonition type="note">
+  The [Dashboard](https://supabase.com/dashboard/project/_) is a transactional environment and will
+  not sustain a session
+</Admonition>
 
-We don't currently support configuring `pgaudit.log_parameter` because it may log secrets in encrypted columns if you are using [`pgsodium`](/docs/guides/database/extensions/pgsodium) or [Vault](/docs/guides/database/vault).
+Inside a session, by default, pgaudit will log nothing:
 
-You can upvote this [feature request](https://github.com/orgs/supabase/discussions/20183) if you'd like this restriction lifted.
+```sql
+--returns 'none'
+show pgaudit.log;
+```
+
+In the session, you can `SET` the `pgaudit.log` variable to record events:
+
+```sql
+-- log CREATE, ALTER, and DROP events
+SET pgaudit.log = 'ddl';
+
+-- log all CREATE, ALTER, DROP, and SELECT events
+SET pgaudit.log = 'read, ddl';
+
+-- log nothing
+SET pgaudit.log = 'none';
+```
+
+### User Logging
+
+There are some cases where you may want to monitor a database user's actions. For instance, let's say you connected your database to [Zapier](https://supabase.com/partners/integrations/zapier) and created a custom role for it to use:
+
+```sql
+CREATE USER zapier WITH PASSWORD '<new password>';
+```
+
+To ensure reliability, you may want to log all actions initiated by `zapier`. That can be done with the following command:
+
+```sql
+ALTER ROLE zapier SET pgaudit.log TO 'all';
+```
+
+To remove the settings, execute the following code:
+
+```sql
+-- disables role's log
+ALTER ROLE zapier SET pgaudit.log TO 'none';
+
+-- check to make sure the changes are finalized:
+SELECT
+  rolname,
+  rolconfig
+FROM pg_roles
+WHERE rolname = 'zapier';
+```
+
+### Global Logging
 
+<Admonition type="caution">
+  Use global logging sparingly as it can generate millions of logs per hour. Consider alternatives
```

Comment:
```suggestion
  Use global logging sparingly as it can generate a large amount of logs and make it difficult to find specific events of importance. Consider limiting the scope of what is logged or defer to user logging where possible.
```